ci_framework/roles/os_net_setup/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+# assiming having a login session with the right namespace already
+- name: get openstack admin credentials from k8s
+  block:
+  - name: Get clouds.yaml configuration
+    ansible.builtin.command:
+      cmd: oc get configmaps openstack-config -o jsonpath={.data}
```

Comment:
This may not be specific to this PR, but is there a place where we explicitly specify the k8s namespace that this command needs to be run on or does it use the active namespace?

Reply:
ATM it is supposed to use the current session from earlier login, and assumingit is ok.
Just added here until the k8s ansible deps not sorted out.

Reply:
Something similar to [this](https://github.com/openstack-k8s-operators/ci-framework/pull/251/files#diff-2779111184fca9ec9fde71edbce55f3530cd9cdd8dc3d9a3f68ac809240d2276R10) maybe?

Reply:
indeed, we really should be explicit regarding the namespace we want to query.

Reply:
Sorry, thought we are speaking about auth context. ;-)

Thanks.

Reply:
command removed